source/disputes/index.html.md.erb

```diff
@@ -0,0 +1,95 @@
+---
+title: Report on a dispute
+last_reviewed_on: 2022-07-22
+review_in: 6 months
+weight: 65
+---
+
+# Report on a dispute
+
+A dispute is when a paying user challenges a completed payment through their bank.
+
+There are many possible reasons for a dispute. For example, your user might not recognise a payment on their bank statement, or they might believe they’ve paid for your service twice.
+
+GOV.UK Pay will email you whenever a user disputes a payment. Our email will explain the reason for the dispute and what you need to do. You might need to provide evidence that the user needed to make the payment.
+
+If you lose a dispute, you must pay back the transaction amount and a dispute fee set by your payment service provider. You’ll only pay for lost disputes raised on or after 8 August 2022. If you lose a dispute that was raised before 8 August 2022, you will not be charged.
+
+## Search disputes
+
+You can search disputes using the GOV.UK Pay API.
+
+Use the following endpoint to search disputes:
+
+`GET /v1/disputes`
+
+Example response:
+
+```json
+{
+  "page": 1,
+  "count": 20,
+  "total": 20,
+  "results:": [
+    {
+      "amount": 1200,
+      "created_date": "2022-01-21T17:15:000Z",
+      "dispute_id": "geCju5lhLGr0B2jfiT3jo5wDek",
+      "evidence_due_date": "2022-01-10T23:59:59.000Z",
+      "fee": 1500,
+      "net_amount": -2500,
+      "payment_id": "hu20sqlact5260q2nanm0q8u93",
+      "reason": "credit_not_processed",
+      "status": "lost",
+      "settlement_summary": {
+        "settled_date": "2022-01-21"
+      },
+      "_links": {
+        "href": "https://publicapi.payments.service.gov.uk/v1/payments/hu20sqlact5260q2nanm0q8u93",
+        "method": "GET"
+      }
+    }
+  ]
+}
+
+```
+
+Each object in `results` represents a single dispute and contains:
+
+* the amount of the disputed payment in pence (`amount`)
+* when the user raised the dispute (`created_date`)
```

Comment:
I imagine there could be a delay between the user raising the dispute and us getting told about it due to delays with the bank processing it and sending it to Stripe - although I don't know about this. Would it be more accurate to say this is when the bank informed us that the user had raised a dispute? Not sure how much that accuracy matters here though.

Reply:
I think it's worth clarifying. I've changed this to:

> The date and time the user's bank told GOV.UK Pay about this dispute.